samples/deployment/deploy.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+################################################################################
+# This script will do following:
+#   - Deploy OpenAM along with DS (CTS, CFGStore, UsrStore)
```

Comment:
Global change: pls refer to OpenAM as AM. The one exception is if you refer to one of our pods which are called "openam" (LC) (which we really ought to change...)
Sugg: s/CFGStore/config store/ and s/UsrStore/user store/